book/src/sprites/set_up_the_render_pass.md

```diff
@@ -0,0 +1,58 @@
+# Set Up The Render Pass
+
+`Amethyst` supports drawing sprites using the `DrawSprite` render pass. To enable this you have to do the following:
+
+1. Build a `Pipeline` with the `DrawSprite` pass, and use the `.with_transparency(..)` method if your sprites have transparent pixels.
```

Comment:
This sentence is a bit long. I would redo it to:

1. Build a `Pipeline` with the `DrawSprite` pass. If your sprites have transparent pixels use the `.with_transparency(..)` method.